lending/loan_management/doctype/loan/loan.py

```diff
@@ -189,6 +193,33 @@ def validate_cost_center(self):
 			if not self.cost_center:
 				frappe.throw(_("Cost center is mandatory for loans having rate of interest greater than 0"))
 
+	def set_optional_accounts(self):
```

Comment:
I think this validation is not needed since we are enforcing it at the Loan Product stage, and we fetch the value from the loan product